app/models/derby_event.rb

```diff
@@ -43,15 +55,13 @@ def location_html
   end
 
   def rulesets
-    ['mrda', 'wftda', 'jrda', 'made', 'usars', 'rdcl'].map do |ruleset|
-      send(ruleset) ? ruleset.upcase : nil
-    end.compact.join(', ')
+    RULESETS.select{|ruleset| send(ruleset) }
```

Comment:
up-casing is view logic so I moved that to the view, not sure if the jbuilder needs a change too so I might have broken it